Models/Plant/Organs/Root.cs

```diff
@@ -395,7 +395,10 @@ public double WaterTensionFactor
                         double[] pawc = Z.soil.PAWC;
                         Biomass[] layerLiveForZone = Z.LayerLive;
                         for (int i = 0; i < Z.LayerLive.Length; i++)
-                            MeanWTF += layerLiveForZone[i].Wt / liveWt * MathUtilities.Bound(2 * paw[i] / pawc[i], 0, 1);
+                            if (paw[i] < pawc[i] & paw[i] > 0 & pawc[i] > 0)
```

Comment:
Should these bitwise AND operators be logical AND?
```suggestion
                            if (paw[i] < pawc[i] && paw[i] > 0 && pawc[i] > 0)
```